pnpm-lock.yaml

```diff
@@ -1,9 +1,5 @@
 lockfileVersion: '6.0'
 
-settings:
-  autoInstallPeers: true
-  excludeLinksFromLockfile: false
-
```

Comment:
:question: I wonder if I'm running a different `pnpm` version to you, just wondering why these would be removed?

Reply:
I'm running `@latest` (`8.7.6`). Does updating to that resolve this?

Reply:
Looks like bumping my version resolved this :raised\_hands: